py/server/deephaven/pandas.py

```diff
@@ -125,8 +126,8 @@ def to_pandas(table: Table, cols: List[str] = None, dtype_backend: Literal[None,
         cols (List[str]): the source column names, default is None which means include all columns
         dtype_backend (str): Which dtype_backend to use, e.g. whether a DataFrame should have NumPy arrays,
             nullable dtypes are used for all dtypes that have a nullable implementation when “numpy_nullable” is set,
-            pyarrow is used for all dtypes if “pyarrow” is set. default is None, meaning Numpy backed DataFrames with
-            no nullable dtypes.
+            pyarrow is used for all dtypes if “pyarrow” is set. None means Numpy backed DataFrames with no nullable
+            dtypes. Default is "numpy_nullable".
```

Comment:
Docstring does not explain that "numpy_nullable" also allows support for string types, and whatever other voodoo can happen.